playbooks/group_vars/prometheus/public.yml

```diff
@@ -35,6 +35,8 @@ PROMETHEUS_ALERT_DISK_SPACE_THRESHOLD: 80
 PROMETHEUS_ALERT_HTTP_RESPONSE_TIME_THRESHOLD: 1
 PROMETHEUS_ALERT_HTTP_RESPONSE_FRONTEND_ERROR_THRESHOLD: 70
 
+PROMETHEUS_ALERT_OCIM_HUEY_QUEUE_SIZE_THRESHOLD: 200
```

Comment:
@shimulch, @0x29a, since this exporter is used only on one specific server, can we move this to the specific file under the `host_vars` directory of the `ansible-secrets` repository? Since Shimul doesn't have access to that repo, you will have to do that Demid.